core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala

```diff
@@ -202,6 +202,7 @@ class BaseDataSourceTest(val table: String,
     val df = queryDatasourceTiDBWithTable(sortCol, tableName = tblName)
     val tidbResult = seqRowToList(df.collect(), df.schema)
 
+    println(s"running test on table $tblName")
```

Comment:
use log?

Reply:
We use `println` only in our test codebase. It should be ok. Printing this out can help us know whether the test is run or not. In addition, with this info, it helps us identify test failing on which table.

Reply:
![image](https://user-images.githubusercontent.com/5574887/63255480-71242800-c2a8-11e9-9e2a-f1baaeeb7067.png)

better to use log4j